<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKPrestorerToolUiEditor</class>
 <widget class="QWidget" name="SAKPrestorerToolUiEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButtonOk">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Format</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Suffix</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Algorithm</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="SAKTextFormatComboBox" name="comboBoxFormat"/>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Data</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="SAKEscapeCharacterComboBox" name="comboBoxEscape"/>
      </item>
      <item row="3" column="1">
       <widget class="SAKAffixesComboBox" name="comboBoxPrefix"/>
      </item>
      <item row="4" column="1">
       <widget class="SAKAffixesComboBox" name="comboBoxSuffix"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Escape</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Prefix</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="SAKCrcAlgorithmComboBox" name="comboBoxAlgorithm"/>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QCheckBox" name="checkBoxCrc">
        <property name="text">
         <string>Append CRC data</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Start index</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QSpinBox" name="spinBoxStart">
        <property name="maximum">
         <number>1024</number>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>End index</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QSpinBox" name="spinBoxEnd">
        <property name="maximum">
         <number>1024</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEditDescription"/>
      </item>
      <item row="10" column="0" colspan="4">
       <widget class="QLineEdit" name="lineEditData"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SAKEscapeCharacterComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">SAKEscapeCharacterComboBox.hh</header>
  </customwidget>
  <customwidget>
   <class>SAKTextFormatComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">SAKTextFormatComboBox.hh</header>
  </customwidget>
  <customwidget>
   <class>SAKAffixesComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">SAKAffixesComboBox.hh</header>
  </customwidget>
  <customwidget>
   <class>SAKCrcAlgorithmComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">SAKCrcAlgorithmComboBox.hh</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
